framework/src/play/mvc/Controller.java

```diff
@@ -1252,6 +1252,17 @@ private static void verifyContinuationsEnhancement() {
         }
     }
 
+
+    /**
+     * Checks if the classname is from the jdk, sun or play package and therefore should not be checked for enhancement
```

Comment:
Sorry for being late to the party but this comment feels counter intuitive.  It states that if the class name package is jdk, sun or play then it should NOT be checked for enhancement. As the method returns true if any of those packages match then shouldn't the method be shouldNotBeCheckedForEnhancement() ? Or am I having a brain fart here :)

Reply:
Yes, good point. I already fixed it in master branch.